src/app/interfaces/user.ts

```diff
@@ -0,0 +1,31 @@
+interface Attributes {
+    preferences: object;
```

Comment:
`Object` is very generic, for example any function is an `object`, array is an `object`.
Can you use { [attribute: string]: someType }?